pkg/server/router/manifest.go

```diff
@@ -41,6 +41,8 @@ func NewManifestRouter(s svcframework.Service) (*ManifestRouter, error) {
 // CreateManifestRequest is the request body for creating a manifest, which populates all remaining fields
 // and builds a well-formed manifest object.
 type CreateManifestRequest struct {
+	ManifestName           *string                          `json:"name,omitempty"`
+	ManifestDescription    *string                          `json:"description,omitempty"`
```

Comment:
Consider this as the manifest bit is implied. 
```suggestion
	Name           *string                          `json:"name,omitempty"`
	Description    *string                          `json:"description,omitempty"`
```